src/io/input.jl

```diff
@@ -69,40 +69,102 @@ function readFlowFrame(filename::String)
 end
 
 """
-    loadData(idx, fn, md,panel; method = "asinh", cofactor = 5,
-            reduce = true, sort = true)
+    loadData(dataPath, data, nWorkers; panel=Nothing(), 
+            type = "fcs", method = "asinh", cofactor = 5,
+            reduce = false, sort = false, transform = false)
+
+This function is of 2 parts. Part 1: Generates the temporary binaray files to be loaded by the
+    workers. The Input data will be equally divided into n parts according to the number of workers.
+    Part2: each worker loads independently its own data-package in parallel and returns
+
+# Arguments:
+- `dataPath`: worker index
+- `data`: single filename::String or a metadata::DataFrame with a column sample_name
+- `panel`: Panel table with a column for Lineage Markers and one for Functional Markers,
+    or Array::{Int} used as column indicies, default: Nothing()
```

Comment:
why `Nothing()`?

Reply:
according to Julia documentation this is used for NULL, and like this I can check if the parameter is set or not